src/chunk_list.cpp

```diff
@@ -51,6 +51,21 @@ chunk_t *chunk_get_tail(void)
 }
 
 
+/**
+ * \brief Searches for a non-vbrace chunk
+ *
+ * follows a chunk list until it finds a non-vbrace chunk.
+ * The search direction is determined by parameter dir.
+ *
+ * @return pointer to the found chunk or NULL of none found
+ */
+static chunk_t *chunk_get_nvb(chunk_t           *cur,           /**< [in] chunk to start with*/
```

Comment:
The problem with @param is that the comment and the code will diverge with time and thus the documentation gets wrong. Besides it adds unnecessary redundancy. But my solution works much better if the parameter may be written on a new line which is currently prohibited by the uncrustify rules.